toxcore/group.c

```diff
@@ -29,24 +29,80 @@
 
 #include "util.h"
 
-/* return 1 if the groupnumber is not valid.
- * return 0 if the groupnumber is valid.
- */
-static uint8_t groupnumber_not_valid(const Group_Chats *g_c, int groupnumber)
+typedef ptrdiff_t aint;
+
+#define CLOSEST( i ) ((g->closest_peers_entry & (1ull << i)) != 0)
```

Comment:
Pass `g` to the macro to make it independent of the current lexical environment, and to clarify at the call site which values are used in the operation.